jgiven-gradle-plugin/src/main/java/com/tngtech/jgiven/gradle/internal/JGivenReportsContainerImpl.java

```diff
@@ -11,7 +14,22 @@
 
     public JGivenReportsContainerImpl( Task task ) {
         super( JGivenReport.class, task, NOOP);
-        add(JGivenHtmlReportImpl.class, task);
+        System.out.println("DEBUG:");
+        System.out.println(task.toString());
+        System.out.println("DEBUG: inst=" + this.getInstantiator().toString());
+        JGivenReport report = this.getInstantiator().newInstance(JGivenHtmlReportImpl.class, task);
+        System.out.println("DEBUG: created report");
+        String name = report.getName();
+        System.out.println("DEBUG: got report name");
+        if (name.equals("enabled")) {
+            throw new InvalidUserDataException("Reports that are part of a ReportContainer cannot be named 'enabled'");
+        } else {
+            System.out.println("DEBUG: evaluated name");
+            this.getStore().add(report);
+            this.index();
+        }
```

Comment:
this code is a direct copy from the `add(JGivenHtmlReportImpl.class)` method. I could track the failure to line 20. I.e. where the new class is instantiated